src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01-browser.md

```diff
@@ -22,28 +26,178 @@ export default function () {
 - 🚧 [browser.newBrowserCDPSession()](#browser-newbrowsercdpsession)
 - [browser.newContext([options])](#browser-newcontext-options)
 - [browser.newPage([options])](#browser-newpage-options)
-- 🚧 [browser.on()](#browser-on)
+- 🚧 [browser.on('disconnected')](#browser-on)
 - ❌ [browser.startTracing()](#browser-starttracing)
 - ❌ [browser.stopTracing()](#browser-stoptracing)
 - [browser.version()](#browser-version)
 
-
 ## browser.close()
 
+Closes the browser and all of its pages (if any were opened).
+
+The [Browser](01-browser.md) object itself is considered to be disposed and cannot be used anymore.
+
 ## browser.contexts()
 
+Returns an array of all open browser contexts. In a newly created browser, this will return zero browser contexts.
+
+### Returns
+
+| Type  | Description                                           |
+| ----- | ----------------------------------------------------- |
+| Array | Array of [BrowserContext](../browsercontext/) objects |
```

Comment:
We probably shouldn't use relative links. We rarely use them elsewhere, and they will make updating them more difficult later. I get the benefits in general (shorter, safe from breaking if lower hierarchy changes), but we shouldn't deviate from the standard.

Reply:
https://github.com/grafana/k6-docs/pull/642/commits/5793d753f2b5606307c96f039c3ddbb019bc550b